homeassistant/components/haiku/translations/en.json

```diff
@@ -0,0 +1,14 @@
+{
+    "config": {
+        "abort": {
+            "no_devices_found": "[ERROR]: No devices found",
+            "single_instance_allowed": "[ERROR]: Single instance allowed"
+        },
+        "step": {
+            "confirm": {
+                "description": "The discovery takes about 6 seconds. Would you like to continue? "
```

Comment:
Please run the translation develop script to sync the English translation.

Reply:
What is the script name?

Reply:
```
python -m script.translations develop
```